package/doc/sphinx/source/index.rst

```diff
@@ -59,6 +59,19 @@ members agree and adhere to --- please read it.
 
 .. _installation-instructions: 
 
+
+User Guide
+==========
+The MDAnalysis `userguide`_ provides comprehensive information on how to
```

Comment:
I'd write it *User Guide* consistently, at least that's how https://userguide.mdanalysis.org/stable/index.html refers to itself.
```suggestion
The MDAnalysis `User Guide`_ provides comprehensive information on how to
```